CMakeLists.txt

```diff
@@ -322,20 +307,21 @@ set(AI_LIBS
         PRIVATE roboteam_utils
         PRIVATE roboteam_proto
         PRIVATE rtt::bt
+        PRIVATE rtt::control
         PRIVATE ${SDL2_LIBRARY}
         )
 
 set(AI_INCLUDES
         INTERFACE include
-        PRIVATE include/roboteam_ai
+        PUBLIC include/roboteam_ai
         PRIVATE ${SDL2_INCLUDE_DIRS}
         PRIVATE ${SDL2IMAGE_INCLUDE_DIRS}
         )
 
 # Main Executable
 add_executable(roboteam_ai src/roboteam_ai.cpp ${AI_SOURCES})
 target_link_libraries(roboteam_ai ${AI_LIBS})
-target_include_directories(roboteam_ai ${AI_INCLUDES})
+include_directories(roboteam_ai ${AI_INCLUDES})
```

Comment:
No. This is literally the worst cmake command man has ever created.

Reply:
I changed that because the includes wouldn't work in submodule, will revert, good catch